MFE/common/src/components/Dialogs/WarningDialog/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import {
+  Button,
+  Dialog,
+  DialogTitle,
+  DialogActions,
+  DialogContent,
+  Typography,
+} from '@material-ui/core';
+import { Warning, SubdirectoryArrowRight } from '@material-ui/icons';
+import PropTypes from 'prop-types';
+import useStyles from './style';
+
+const WarningDialog = ({ isOpen, message, cancelButtonText, handleClose, devices }) => {
```

Comment:
I think that `WarningDialog` is not a good name because it is a generic name, meaning that you can use this modal for every warning, but this modal is specialized in devices warning/errors.